lint_tuva_project.sh

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/env bash
```

Comment:
Can you add a comment explaining where/when/how this bash script is used? I see it's used in the github actions, but it would be great to have some kind of overview here as well.

Reply:
Looks great. Thank you!